doc/tutorial/fcm_fundamentals

```diff
@@ -0,0 +1,859 @@
+            associated with the repository they can read from, write to or execute code 
+            from the repository.
+            <br/>
+            This allows for collborations therefore encouraging sharing, developing and 
+            using code for a common purpose.
+            <br/>
+            One of the key aspect of a version control system is that it records the 
+            incremental repository changes.
+            <br/>
+            The version control system remembers/records each version change and who made 
+            the change to the repository.
+            <br/>
+            As version control system record each change and not just the last (most 
+            recent) then a user can work with a specified version of the code from the 
+            repository.
+            <br/>
+            Therefore a user does not need to always work with the most recently submitted 
+            (checked in or committed) code but can work with an alternative version, which 
+            can be helpful for comparisons of changes with older repository versions of 
+            the same code.
+            <br/>
+            This functionality also allows users to explore what the repository contained 
+            at a point in time and who has made changes at what time to the repository.
+        </ul>
+        </p>
+        <h1><a id="S3">Scenarios</a></h1>
+        <p>
+        <ul>
+            <div class="page-header">
+                <h3>What, why, how, who?</h3>
+            </div>
+            <div class="well">
+            Scenario 1
+            <br/>
+            Nicholas, Danny and Dorris are all working on a new project where they need to develop joint code and a joint webpage.
+            <br/>
+            Nicholas and Danny want to work on a file one at a time and then pass around the file to the next person when they have finished the changes they want to make at that moment in time.
+            <br/>
+            Dorris wants to use a version controlled repository so that they can all work on the code together and therefore not have to wait for the previous person to have finished.
+            <br/>
+            <div class="panel-group" id="accordion">
+                <div class="panel panel-default">
+                    <div class="panel-heading">
+                        <h4 class="panel-title">
+                            <a data-toggle="collapse" data-parent="#accordion" href="#collapseOne">
+                            <span class="glyphicon"></span>
+                            What other pros and cons can you think of for each of the ways of working?
+                            </a>
+                        </h4>
+                    </div>
+                    <div id="collapseOne" class="panel-collapse collapse">
+                        <div class="panel-body">
+                            Nicholas and Danny pros
```

Comment:
Could you put a colon on the end of the line or bold the text or something like that to highlight this is a heading (makes it easier to read the flow of pros/cons